experimental/traffic-portal/src/app/core/cache-groups/regions/table/regions-table.component.ts

```diff
@@ -94,21 +94,26 @@ export class RegionsTableComponent implements OnInit {
 			name: "Delete"
 		},
 		{
-			action: "viewRegions",
+			action: "viewDivision",
 			multiRow: false,
-			name: "View Regions"
+			name: "View Division"
```

Comment:
Links should be used for navigation, not click handlers. Instead of a `ContextMenuAction`, this should be a `ContextMenuLink` that has an `href` function that constructs the requisite link from the input selected Region.